
test_tricycle_controller:
  ros__parameters:
    traction_joint_name: traction_joint
    steering_joint_name: steering_joint
    wheel_radius: 0.125
    wheelbase: 1.252
    use_stamped_vel: false
    enable_odom_tf: false
    use_sim_time: false
    pose_covariance_diagonal: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    twist_covariance_diagonal: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    odom_only_twist: true
    publish_ackermann_command: true
    traction:
      max_acceleration: 5.0
      max_deceleration: 8.0
    steering:
      max_position: 1.57 # pi/2
      max_velocity: 1.0
